test/parallel/test-os.js

```diff
@@ -191,3 +181,20 @@ is.string(pwd.username);
 assert.ok(pwd.homedir.includes(path.sep));
 assert.strictEqual(pwd.username, pwdBuf.username.toString('utf8'));
 assert.strictEqual(pwd.homedir, pwdBuf.homedir.toString('utf8'));
+
+// Test that the Symbol.toPrimitive functions work correctly
+/* eslint-disable no-restricted-properties */
+[
+
+  [assert.equal, `${os.freemem}`, os.freemem()],
```

Comment:
Nit: Maybe better as `assert.strictEqual, `${os.freemem}`, `${os.freemem()}`] here and similarly below? Matter of style preference, I suppose.